src/content/overview/fleet-management/app-management/index.md

```diff
@@ -0,0 +1,135 @@
+  version: "0.7.2"
+  kubeConfig:
+     inCluster: false
+  userConfig:
+    configMap:
+      name: "kong-user-values"
+      namespace: "x7jwz"
+```
+
+Below you can see a high level overview of the components and resources that work together to enable the features of the Giant Swarm app platform:
+
+![A diagram showing an overview of various components and concepts that make up the Giant Swarm App Platform](app-platform-overview.png)
+<!-- Original version: https://docs.google.com/drawings/d/1V3KcUImxRdrrb2v_nIQnkapHiRkRM6t8PoYGCqWebYY/edit -->
+
+#### The Giant Swarm app catalog
+
+This catalog contains our stable, fully managed apps, with SLA (for example the ingress nginx controller).
+
+The maturity levels of apps in this catalog are expressed through semantic versioning as follows:
+
+- Version with `-alpha` or `-beta` suffix - the application is only at a basic maturity level. There is no stable release. It's supported on a best effort basis,
+- Version with `-rc*` suffix - the application is at a preview maturity level. This allows customers to preview a new release of an application and evaluate new features. It's supported on a best effort basis.
+- Version >= `v1.0.0` with no suffix - the specified version of the application is at a stable maturity level. It's available to our customers as a managed offering with support and SLA.
+
+### Managed app definition
+
+A `managed app` is an app in our Giant Swarm catalog that provides:
+
+- Safe and tested deployment
+
+The `Helm` chart is ready to use and tested, either sanitizing the upstream forked applications or creating good defaults for our maintained ones. [It's offered a common way of building apps](https://github.com/giantswarm/app-build-suite) and a [testing framework](https://github.com/giantswarm/app-test-suite) which ensure the application is deployable and works as expected. Security and upgrade viability are checked too during the integration process.
+
+- Monitoring
+
+Giant Swarm makes sure all the main components of the app are running and that the app is working as expected. At the same time, our automation set up monitoring and alerting on necessary metrics to ensure our service level agreements.
+
+In case of an alert, operations team perform an Root Cause Analysis (RCA) to understand if it's a Giant Swarm or customer-inflicted issue that broke the application.
+
+__Note__: Giant Swarm don't fix bugs upstream when they involve significant code changes. That said, our team always try to find a workaround or the root cause of the issue and submit a ticket to the upstream project. In some cases, our engineers fix the bug ourselves where it’s necessary and possible for us, and provide the fix to the upstream project. This might result in temporarily running a non-upstream patch version from Giant Swarm until upstream merges our patch. Customers can, in general, expect the same level of service for a managed optional app as they get with "default" apps such as `coredns`.
```

Comment:
```suggestion
__Note__: In general, Giant Swarm does not fix bugs in upstream projects, however our team will always try to find a solution to an issue, or the root cause of the issue and submit a ticket to the upstream project when relevant to do so. In certain instances, our engineers may attempt to fix the bug where it is necessary and possible for us, and provide the fix to the upstream project. This can result in Giant Swarm temporarily running an intermediate patch version whilst waiting for upstream to release a new version containing our patch. Customers can expect the same level of service for a managed optional app as they get with "default" apps such as `coredns`.
```